frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,22 @@
+import { type IconNames } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { iconNameToPlainSvgMap } from '#libs/maps/icon-name-to-plain-svg.map.js';
+import { type ClassValue } from '#libs/types/types.js';
+
+type Properties = {
+  name: IconNames;
+  className?: ClassValue;
+};
+
+const Icon: React.FC<Properties> = ({ name, className = '' }) => {
+  const SelectedIcon = iconNameToPlainSvgMap[name];
+  const classNames = getValidClassNames(className as string);
```

Comment:
I don't think we need to use getValidClassNames here. Just use className that comes from props